lib/utils.js

```diff
@@ -156,13 +118,13 @@ exports.forceHttps = function (req, res, next) {
 // would look for /app/views/test.html
 // and /app/views/test/index.html
 
-function renderPath (path, res, next) {
-  // Try to render the path
+async function renderPath (path, res, next) {
   const model = {}
   if (path === 'index') {
     model.serviceNameFileLocation = 'app/config.json'
   }
-  res.render(path, model, function (error, html) {
+  // Try to render the path
+  res.render(path, model, async function (error, html) {
```

Comment:
I didn't think that render can take an async function, at least, I can't find it documented anywhere 😟 At yet it does seem to work 🤔

Reply:
This article seems to suggest that Express doesn't fully support async. Handlers can be asynchronous and it will work for the happy path, but if any promises are rejected the server will hang. https://ioannisioannou.me/using-async-await-in-express-js/.

I feel like using async code here might be a bit of a code smell.

Reply:
Removed async